packages/grafana-runtime/src/utils/sessionStorage.ts

```diff
@@ -0,0 +1,11 @@
+export const getReturnToPrevious = () => {
+  return {
+    title: sessionStorage.getItem('returnToPreviousTitle') || '',
+    href: sessionStorage.getItem('returnToPreviousHref') || '',
```

Comment:
I think we should store this in session storage as a single JSON object, to prevent the two properties from getting out of sync, like one property getting cleared and the other one persisting for some reason. These properties only make sense together, so they should be stored together.

Also, I don't think it's a great practice to to default a value to empty string - either it has a real value (a title to display), or it's undefined. This check should go away if we store it as a JSON object - either the object is there with all the properties, or its not.

Reply:
why are we storing this in session storage?